core/storage/feedback/gae_models_test.py

```diff
@@ -79,6 +80,52 @@ def test_raise_exception_by_mocking_collision(self):
                 feedback_thread_model_cls.generate_new_thread_id(
                     'exploration', 'exp_id')
 
+    def test_export_data_trivial(self):
+        user_data = feedback_models.GeneralFeedbackThreadModel.export_data(
+            'fake_user')
+        test_data = {}
+        self.assertEqual(user_data, test_data)
+
+    def test_export_data_nontrivial(self):
+        # Set up testing varibles.
+        test_export_entity_type = feconf.ENTITY_TYPE_EXPLORATION
+        test_export_entity_id = 'exp_id_2'
+        test_export_author_id = 'user_1'
+        test_export_status = 'open'
+        test_export_subject = 'dummy subject'
+        test_export_has_suggestion = True
+        test_export_summary = 'This is a great summary.'
+        test_export_message_count = 0
+
+        feedback_thread_model = feedback_models.GeneralFeedbackThreadModel(
+            entity_type=test_export_entity_type,
+            entity_id=test_export_entity_id,
+            original_author_id=test_export_author_id,
+            status=test_export_status,
+            subject=test_export_subject,
+            has_suggestion=test_export_has_suggestion,
+            summary=test_export_summary,
+            message_count=test_export_message_count
+        )
+
+        feedback_thread_model.put()
+
+        user_data = (feedback_models
```

Comment:
Here and elsewhere: if the interior of a parenthetical spans more than a line, break after the opening '('.

Reply:
Done

Reply:
Not done. Break after the opening '('.